### PR TITLE
chore(PE-5886): fix docker-compose for integration tests against local s…

### PR DESCRIPTION
**Pull request by dtfiedler**
…erivces

Fixing these so we can pull in `arlocal` and test interactions related to AR credit payments

Steps to validate it's working
- Create a `.env` file and add the STRIPE_SECRET_KEY from the `test` environment (`sk_test_XXX`)
- `yarn test:docker`

It should spin up the upload and payment service locally, and pass all tests.

### CODE REVIEW

File: Dockerfile.test
Comment (dtfiedler) [inline Dockerfile.test:1]: we don't need this anymore - we'll run the tests directly against the local services

File: tests/wallets/ByQEA5jhJvzlhfI4sFgB23kjGpxDK6OIE0i3sSnmTGU.json
Comment (dtfiedler) [inline tests/wallets/ByQEA5jhJvzlhfI4sFgB23kjGpxDK6OIE0i3sSnmTGU.json:1]: we inject two wallets to the upload service so we can validate tests. one is the upload-service wallet used to sign receipts, the other is the allow listed wallet to properly test uploads

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**